src/data/data.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "src": "https://global-uploads.webflow.com/63994dae1033718bee6949ce/649f3f286772ee18775d26f1_mby979eTPSDlRpr7H2MNSly1Sk-p-500.png",
+    "alt": "Drippi.ai",
+    "title": "Drippi.ai",
+    "description": "A tool to automate personalized Twitter outreach.",
```

Comment:
maybe consider more data points based on what sorting options you have ? (sort by recents?)

Reply:
cant we just sort recents using the ID? And we can sort Alphabetically using the title. maybe add tags for Filter